keyboards/chosfox/cf81/cf81.c

```diff
@@ -0,0 +1,209 @@
+    {0, D_14,  E_14,  F_14},
+    {1, D_2,   E_2,   F_2},
+
+    {0, G_1,   H_1,   I_1},
+    {0, G_2,   H_2,   I_2},
+    {0, G_3,   H_3,   I_3},
+    {0, G_4,   H_4,   I_4},
+    {0, G_5,   H_5,   I_5},
+    {0, G_6,   H_6,   I_6},
+    {0, G_7,   H_7,   I_7},
+    {0, G_8,   H_8,   I_8},
+    {0, G_9,   H_9,   I_9},
+    {0, G_10,  H_10,  I_10},
+    {0, G_11,  H_11,  I_11},
+    {0, G_12,  H_12,  I_12},
+    {0, A_15,  B_15,  C_15},
+    {0, G_13,  H_13,  I_13},
+    {1, D_3,   E_3,   F_3},
+
+    {0, J_1,   K_1,   L_1},
+    {0, A_16,  B_16,  C_16},
+    {0, J_2,   K_2,   L_2},
+    {0, J_3,   K_3,   L_3},
+    {0, J_4,   K_4,   L_4},
+    {0, J_5,   K_5,   L_5},
+    {0, J_6,   K_6,   L_6},
+    {0, J_7,   K_7,   L_7},
+    {0, J_8,   K_8,   L_8},
+    {0, J_9,   K_9,   L_9},
+    {0, J_10,  K_10,  L_10},
+    {0, J_11,  K_11,  L_11},
+    {0, J_12,  K_12,  L_12},
+    {1, D_7,   E_7,   F_7},
+
+
+    {0, J_13,  K_13,  L_13},
+    {0, J_14,  K_14,  L_14},
+    {0, J_15,  K_15,  L_15},
+    {0, J_16,  K_16,  L_16},
+
+    {0, G_14,  H_14,  I_14},
+    {0, G_15,  H_15,  I_15},
+    {0, G_16,  H_16,  I_16},
+
+    {0, D_15,  E_15,  F_15},
+    {1, D_6,   E_6,   F_6},
+    {1, D_5,   E_5,   F_5},
+
+
+    {1, G_1,   H_1,   I_1},
+    {1, G_2,   H_2,   I_2},
+    {1, G_3,   H_3,   I_3},
+    {1, G_4,   H_4,   I_4},
+    {1, G_5,   H_5,   I_5},
+    {1, G_6,   H_6,   I_6},
+    {1, G_7,   H_7,   I_7},
+    {1, G_8,   H_8,   I_8},
+    {1, G_9,   H_9,   I_9},
+    {1, G_10,  H_10,  I_10},
+    {1, G_11,  H_11,  I_11},
+
+    {1, J_1,   K_1,   L_1},
+    {1, J_2,   K_2,   L_2},
+    {1, J_3,   K_3,   L_3},
+    {1, J_4,   K_4,   L_4},
+    {1, J_5,   K_5,   L_5},
+    {1, J_6,   K_6,   L_6},
+    {1, J_7,   K_7,   L_7},
+    {1, J_8,   K_8,   L_8},
+    {1, J_9,   K_9,   L_9},
+    {1, J_10,  K_10,  L_10},
+    {1, J_11,  K_11,  L_11},
+};
+// clang-format on
+bool rgb_matrix_indicators_advanced_kb(uint8_t led_min, uint8_t led_max) {
+    if (!rgb_matrix_indicators_advanced_user(led_min, led_max)) {
+        return false;
+    }
+    if (host_keyboard_led_state().caps_lock) {
+        RGB_MATRIX_INDICATOR_SET_COLOR(44, 255, 255, 255);
+    }
+    if (keymap_config.no_gui) {
+        RGB_MATRIX_INDICATOR_SET_COLOR(74, 255, 255, 255);
+    }
+    return true;
+}
+
+#endif
+
+enum __layers {
+    WIN_B,
+    WIN_FN,
+    MAC_B,
+    MAC_FN
+};
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    if (!process_record_user(keycode, record)) {
+        return false;
+    }
+    switch (keycode) {
+        case DF(WIN_B):
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(WIN_B);
+                layer_state_set(1<<WIN_B);
+            }
+            return false;
+        case DF(MAC_B):
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(MAC_B);
+                layer_state_set(1<<MAC_B);
+                keymap_config.no_gui     = 0;
```

Comment:
Any specific reason for disabling win/gui/cmd on the mac layer?  The key is still usable and used heavily